client/src/assessment.ts

```diff
@@ -26,7 +34,27 @@ export interface GetAssessmentsForResourceInput {
 export interface AssessmentWithDimensionAndResource {
     assessment: Assessment,
     dimension: Option<Dimension>,
-    resource: Option<Record>
+    resource: Option<HoloRecord>
+}
+
+export type AssessmentEh = EntryHash
+
+export const getAssessmentKeyValues = (returnMap: MapAssessmentsByHash): [EntryHashB64, Assessment][] => {
+    return Object.entries(returnMap);
+}
+
+export const getAssessmentValue = (returnMap: MapAssessmentsByHash): Assessment | undefined => {
+    return Object.values(returnMap).pop();
+}
+
+export const getAssessmentKey = (returnMap: MapAssessmentsByHash): EntryHashB64 | undefined => {
+    return Object.keys(returnMap).pop();
+}
+
+export const getResourceHash = (a: Assessment): EntryHashB64 => {
+    return encodeHashToBase64(a.resource_eh);
```

Comment:
As discussed, when I merge in #41 I will probably replace the core of `SensemakerService` such that all hash identifiers are encoded to strings at the API boundary.
It'll be easy enough to track down these additional encoding & decoding passes when the time comes.

Reply:
Oh, that's right. And I probably also *also* won't need to do that at all since you're converting to use of `HoloHashB64` internally in the zome API interfaces.

We also discussed providing 'raw' versions of these APIs as alternative interfaces for direct RPC communication between Holochain Cells with less de/serialization overhead.